content/en/user-guide/aws/opensearch/index.md

```diff
@@ -203,6 +203,52 @@ The following output will be visible on your terminal:
 
 It's important to note that any unauthorized requests will yield an HTTP response with a status code of 401 (`Unauthorized`).
 
+## OpenSearch Dashboards
```

Comment:
It would be nice to have a link to the tool here as well

Reply:
Thanks, good point, I converted the first mention in the text to a link to the official docs: https://opensearch.org/docs/latest/dashboards